src/core/auth/authentication/pages/LogoutPage.tsx

```
@@ -35,7 +35,12 @@ const LogoutPage = () => {
     return () => {};
   }, [logoutUrl, getLogoutUrl]);
 
-  if (error) return <Typography>{error}</Typography>;
+  if (error)
+    return (
+      <Typography>
+        <>{error}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)